README.adoc

```diff
@@ -215,6 +215,19 @@ list of available Rake tasks via:
 $ bundle exec rake -T
 ---------------------
 
+=== Code style
+
+As most other open-source projects vagrant-service-manager has a set of conventions
```

Comment:
Comma needed after "As most other open-source projects" as doesn't read well in continuous with "vagrant-service-manager"